doc/admin/deployment.md

```diff
@@ -68,6 +68,10 @@ $ sudo grub2-mkconfig --output=/boot/grub2/grub.cfg
 $ sudo reboot
 ```
 
+!!! warning
+VFIO support is currently planned for DAOS 1.2 and won't work until we move to
+SPDK 20.01.1
```

Comment:
need 4 spaces before VFIO and the of the block to be aligned on that otherwise Mkdocs does not pick it up.